### PR TITLE
Use Rust for the front-end too

### PR DESCRIPTION
**Pull request by BaptisteGelez**
Rust can compile to WASM, so let's use it for front-end code as well.

To compile the front-end:

```
cargo install cargo-web
cd plume-front
cargo web deploy -o ../static
```

### CODE REVIEW

File: static/plume-front.js [hunk 1]
```diff
@@ -0,0 +1,572 @@
+"use strict";
```
Comment (fdb-hiroshima) [inline static/plume-front.js]: I think this file souldn't be committed, it's auto generated and things like `__cargo_web_snippet_199d5eb25dfe761687bcd487578eb7e636bd9650` are probably not reproducible between builds

**Comment by fdb-hiroshima** [comment]
@igalic considering our use-case, probably not much. The main advantage is the whole team being able to debug it I think

**Comment by BaptisteGelez** [comment]
There may be better performances once with the future editor, especially when writing big articles. Another advantage is that we will be able to share code between backend and frontend if we need it.

**Comment by fdb-hiroshima** [comment]
@BaptisteGelez I see Travis failed because of cargo-web installation, you might be interested into this https://github.com/koute/cargo-web#using-cargo-web-on-travis .

File: .gitignore [hunk 2]
```diff
@@ -17,3 +17,6 @@ tags.*
 !tags.rs
 search_index
 main.css
+static/index.html
```
Comment (fdb-hiroshima) [inline .gitignore]: this line is no longer required

**Pull request merged**